normandy/recipes/api.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import permissions, viewsets
+
+from normandy.recipes.models import Action
+from normandy.recipes.serializers import ActionSerializer
+
+
+class ActionViewSet(viewsets.ModelViewSet):
+    """Viewset for viewing and uploading recipe actions."""
+    queryset = Action.objects.all()
+    serializer_class = ActionSerializer
+    permission_classes = [permissions.DjangoModelPermissions]
+
+    lookup_field = 'name'
+    lookup_value_regex = r'[_\w]+'
```

Comment:
I think the default for this value is `r'[^/\.]'`, which is a superset of this regex. Is there a reason to restrict this down?